docs/package-imports.md

```diff
@@ -0,0 +1,41 @@
+An agent that is generated from the AEA framework is a modular system with different skills, connections, and protocols.
+When we create a new agent with the command `aea create my_aea` we create the file structure that looks like the following :
+- connections
+- protocols
+- skills
+- vendor
+- aea-config.yaml
+
+In all the folders except the `vendor`, we can locate the packages we developed. On the other hand, if we download a 
+package from the registry it will be located inside the `vendor's folder` that follows a similar structure.
+
+To use a package, the path for the package must be listed inside the `aea-config.yaml` file.
```

Comment:
To use a package, the `public_id` for the package must be listed in the `asa-config.yaml` file.